contrib/imgui/imgui.cpp

```diff
@@ -7169,7 +7169,8 @@ bool ImGui::Begin(const char* name, bool* p_open, ImGuiWindowFlags flags)
         {
             if (!window->ViewportOwned && viewport_rect.GetWidth() > 0 && viewport_rect.GetHeight() > 0.0f)
             {
-                ClampWindowPos(window, visibility_rect);
+                if (!GetIO().ImGuiIgnoreClampWindow)
```

Comment:
You can use `g.IO` here instead of `GetIO()`

Reply:
fixed.